server/sonar-web/src/main/js/app/components/nav/app.js

```diff
@@ -24,13 +24,13 @@ import ReactDOM from 'react-dom';
 import GlobalNav from './global/global-nav';
 import ComponentNav from './component/component-nav';
 import SettingsNav from './settings/settings-nav';
-import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../api/nav';
+import { getGlobalNavigation, getComponentNavigation, getSettingsNavigation } from '../../../api/nav';
 
 export default class App {
   start () {
     const options = window.sonarqube;
 
-    require('../../components/workspace/main');
+    require('../../../components/workspace/main');
 
     return new Promise((resolve) => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove parentheses around the parameter of this arrow function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)